.eslintrc

```diff
@@ -30,15 +30,6 @@
      */
     "class-methods-use-this": ["error", { "exceptMethods": ["run"] }],
 
-    "import/order": ["error", {
```

Comment:
Moved into our core standards and tagged a major release of that pkg. https://github.com/readmeio/standards/commit/edffdbe631ca3927b882fd19d8f03dd49451d937